config/dev.secret.exs.sample

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+config :ueberauth, Ueberauth.Strategy.Google.OAuth,
+  client_id: GOOGLE_CLIENT_ID,
+  client_secret: GOOGLE_SECRET
```

Comment:
We can do that but is it really that important? I mean there are no REALLY secret data there that anyone could use (unless they get access to your computer, then you are fucked) 😄